rmw_fastrtps_shared_cpp/test/test_guid_utils.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtest/gtest.h"
+
+#include "fastrtps/rtps/common/Guid.h"
+
+#include "rmw_fastrtps_shared_cpp/guid_utils.hpp"
+
+using rmw_fastrtps_shared_cpp::copy_from_byte_array_to_fastrtps_guid;
+using rmw_fastrtps_shared_cpp::copy_from_fastrtps_guid_to_byte_array;
+
+static constexpr size_t byte_array_size =
```

Comment:
Out of curiosity, what does byte_array_size evaluate to?

Reply:
It's 16, but I preferred not to hard-code it.